Artesian/Artesian.SDK/Factory/IMarketData.cs

```diff
@@ -110,6 +112,33 @@ public interface IMarketData
         /// </remarks>
         /// <returns> Marketdata </returns>
         IBidAskWritable EditBidAsk();
+
+        /// <summary>
+        /// Update Derived Configuration
+        /// </summary>
+        /// <param name="derivedCfg">The Derived Configuration Muv to be updated</param>
+        /// <param name="force">Force the update of configuration also if another rebuild process is running (Defualt=false)</param>
+        /// <param name="ctk">Cancellation Token</param>
+        /// <returns></returns>
+        Task UpdateDerivedConfiguration(DerivedCfgMuv derivedCfg, bool force, CancellationToken ctk = default);
```

Comment:
I'm sure Backend defines 'force' with default 'false'.

That should be here too.

Could ve best to have 2 overloads, so in case force is not needed but ctk is (should) the ctk can be passed without having to define the force parameter.

Update(cfg,ctk) => Update(cfg,false,ctk);
Update(cfg,force,ctk)